bitnami/common/templates/_images.tpl

```diff
@@ -17,28 +17,27 @@ Return the proper image name
 
 {{/*
```

Comment:
Ah, that's the idea. This changes the behavior completely, right?
Initially `.global.imagePullSecrets` was more like override the pullSecrets of a pod completely and now the global pullSecrets are appended to the individual pod pullSecrets (`.image.pullSecrets`).

Reply:
Yes, in the end, it doesn't matter I mean we can have all of them there if we have the right one to pull the image at this point. Even it is better because we can have just one helper to be used on all of them.

```yaml
{{/*
Return the proper Docker Image Registry Secret Names
*/}}
{{- define "contour.imagePullSecrets" -}}
{{- include "common.images.pullSecrets" (dict "images" (list .Values.contour.image .Values.envoy.image) "global" .Values.global) -}}
{{- end -}}
```